app/main.py

```diff
@@ -1,3 +1,63 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
```

Comment:
The `__init__` method should handle non-integer inputs gracefully. Consider adding a type check or conversion to ensure `km` is always an integer.